doc/plugins/org.polarsys.capella.re.doc/html/08. Replicable Element (REC-RPL)/8.1. Introduction.html

```diff
@@ -34,6 +34,9 @@ <h2 id="Definitions">Definitions</h2>
 			<li>Constrained Reuse: Internal elements can be added inside a RPL, but 	constraints and Interfaces (Function and Component Ports for 	example) defined in REC cannot be modified.</li>
 			<li>Inheritance: Any element can be added in the RPL, including new 	Interfaces.</li>
 		</ul>
+		<p>More details can be find at &lt;a href="
+			<a href="https://github.com/eclipse/capella-cybersecurity/issues">https://github.com/eclipse/capella-cybersecurity/issues</a>" target="_blank"&gt;Replicate model elements in Capella&lt;/a&gt;&gt;
+		</p>
 		<h2 id="Examples">Examples</h2>
```

Comment:
Provide the link to yt, make sure target is added, as below:
![image](https://user-images.githubusercontent.com/60220371/128974670-707dc3d0-8963-47c3-9b8e-7a58564862f8.png)